lib/native/.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+**/*.rs.bk
+Cargo.lock
```

Comment:
This should be covered by the top-level `.gitignore`, no?

Reply:
Ah, that was copied from `lib/wasm/.gitignore`, which is probably an leftover from when that code lived in its own repo. I'll remove both.